indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -468,7 +468,7 @@
           :personnel m/string-field-mapping
 
           ;; analyzed field for keyword searches
-          :keyword m/text-field-mapping
+          :keyword2 m/text-field-keyword-mapping
```

Comment:
Why did you have to create a keyword2? Why can't we just use keyword? We just need to re-index the collections, which takes 10 minutes and we do that everyday anyway and the old data won't exist anymore.

Reply:
I got your feedback on slack that you can't change the type of data that exists in the keyword.

Reply:
correct, you can't update the mapping of an existing ES field.